apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -611,6 +611,58 @@ export const resources = {
         },
       ],
     },
+    {
+      name: 'Supabase CLI',
+      items: [
+        {
+          name: 'Overview',
+          url: '/guides/resources/supabase-cli',
+          items: [],
+        },
+        {
+          name: 'Local Development',
+          url: '/guides/resources/supabase-cli/local-development',
+          items: [],
+        },
+        {
+          name: 'Managing Environments',
+          url: '/guides/resources/supabase-cli/managing-environments',
+          items: [],
+        },
+      ],
+    },
```

Comment:
@saltcod I assume you didn't mean to add Supabase CLI back to the Resources section.
```suggestion
```

Reply:
aah right! thanks